Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsBuildProvider.cs

```diff
@@ -9,21 +9,31 @@ namespace UnityEngine.InputSystem.Editor
 {
     internal class InputSettingsBuildProvider : IPreprocessBuildWithReport, IPostprocessBuildWithReport
     {
+        InputActionAsset m_ProjectWideActions;
+        Object[] m_OriginalPreloadedAssets;
         public int callbackOrder => 0;
 
         public void OnPreprocessBuild(BuildReport report)
         {
-            if (InputSystem.settings == null)
-                return;
+            m_OriginalPreloadedAssets = PlayerSettings.GetPreloadedAssets();
+            var preloadedAssets = PlayerSettings.GetPreloadedAssets();
 
-            // If we operate on temporary object instead of input setting asset,
-            // adding temporary asset would result in preloadedAssets containing null object "{fileID: 0}".
-            // Hence we ignore adding temporary objects to preloaded assets.
-            if (!EditorUtility.IsPersistent(InputSystem.settings))
+#if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+            EditorBuildSettings.TryGetConfigObject(InputSettingsProvider.kEditorBuildSettingsActionsConfigKey,
```

Comment:
Why do we reference config object here. We don't seem to use m_ProjectWideActions  for anything